test/dynamics/maximum_height_from_velocity_test.py

```diff
@@ -0,0 +1,36 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+)
+from symplyphysics.laws.kinematic import maximum_height_from_velocity as maximum_height_law
```

Comment:
It won't work. Check your code before submission.

Reply:
changed law path